include/random_walks/gaussian_accelerated_billiard_walk.hpp

```diff
@@ -282,16 +331,16 @@ struct GaussianAcceleratedBilliardWalk
         Point _p;
         Point _v;
         NT _lambda_prev;
-        DenseMT _AA;
+        AA_type _AA;
         E_type _L_cov;   // LL' = inv(E)
-        DenseMT _AE;
+        AE_type _AE;
         E_type _E;
         VT _AEA;
         unsigned int _rho;
         update_parameters _update_parameters;
         typename Point::Coeff _lambdas;
         typename Point::Coeff _Av;
-        bool was_reset;
+        BoundaryOracleHeap<NT> _distances_set;
```

Comment:
This is defined in uniform ABW, it should be better if it is defined in a separate file and both walks include it.

Reply:
yeah, I was thinking about that too, any suggestions for that file name?

Reply:
`boundary_oracle_heap`?